bench/irmin-pack/trace_stat_summary_utils.ml

```diff
@@ -116,6 +116,246 @@ let pp_percent ppf v =
     Format.fprintf ppf "%ce%cx" long_repr.[0] long_repr.[4])
   else Format.fprintf ppf "++++"
 
+(** List of tuples of week timestamp, week transaction count, week ops count and
+    week block count.
+
+    There is ~1 block per minute in tezos and 10080 minutes in a week. Hence the
+    content of the last columns.
+
+    Computed using:
+
+    https://api.tzstats.com/series/block.json?collapse=1w&columns=n_tx,n_ops,count *)
+let weekly_stats =
```

Comment:
Can we put this in a different file? e.g. `tezos_history_metrics.ml`. It's a bit odd to have Tezos-specific numbers in the main logic.

Reply:
Sure